javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -297,6 +297,146 @@ public void shouldCharSeqReturnNotNull() {
         assertThat((Iterable<Character>) charSeq("123")).isNotNull();
     }
 
+    @Test
+    public void shouldArrayWithSingleReturnNotNull() {
+        assertThat(array('1')).isNotNull();
+    }
+
+    @Test
+    public void shouldArrayWithVarArgReturnNotNull() {
+        assertThat(array('1', '2', '3')).isNotNull();
+    }
+
+    @Test
+    public void shouldArrayWithIterableReturnNotNull() {
+        assertThat(array(charSeq('1', '2', '3'))).isNotNull();
+    }
+
+    @Test
+    public void shouldArrayWithStreamReturnNotNull() {
+        assertThat(array(charSeq('1', '2', '3').toJavaList().stream())).isNotNull();
```

Comment:
We already have Value.toJavaStream(). But maybe we will integrate better with Java by deprecating it and adding Value.stream() and Value.parallelStream() :-)